core/tests/puppeteer-acceptance-tests/user-utilities/blog-post-admin-utils.js

```diff
@@ -0,0 +1,461 @@
+   * @param {number} limit - The limit of the blog tags.
+   */
+  async setMaximumTagLimitTo(limit) {
+    // These steps are for deleting the existing value in the input field.
+    const tagInputField = await this.page.$(maximumTagLimitInput);
+    await tagInputField.click({ clickCount: 3 });
+    await this.page.keyboard.press('Backspace');
+
+    await this.type(maximumTagLimitInput, limit);
+    await this.clickOn('Save');
+
+    showMessage('Successfully updated the tag limit to ' + limit + '!');
+  }
+
+  /**
+   * This function checks if the tag limit is not equal to.
+   * @param {number} limit - The limit of the blog tags.
+   */
+  async expectMaximumTagLimitNotToBe(limit) {
+    await this.page.evaluate(async(limit) => {
+      const tagLimit = document.getElementById('mat-input-0').value;
+      if (tagLimit.value === limit) {
+        throw new Error('Maximum tag limit is already ' + limit + '!');
+      }
+    }, limit);
+    showMessage('Maximum tag limit is not ' + limit + '!');
+  }
+
+  /**
+   * This function checks if the tag limit is equal to.
+   * @param {number} limit - The limit of the blog tags.
+   */
+  async expectMaximumTagLimitToBe(limit) {
+    await this.page.evaluate(async(limit) => {
+      const tagLimit = document.getElementById('mat-input-0').value;
+      if (tagLimit.value !== limit) {
+        throw new Error('Maximum tag limit is not ' + limit + '!');
+      }
+    });
+    showMessage('Maximum tag limit changed to ' + limit + '!');
+  }
+};
+
+
+/* NOTE 1 */
+/** Giving explicit waitForTimeout when clicking the 'CREATE NEW BLOG POST'
+ * button in the blog dashboard page as we need to add Bio for the user for
+ * the first time we access the /blog-dashboard page. It is a modal dialog
+ * box and beside it the blog dashboard is already loaded (in DOM also), thats
+ * why we cannot wait for 'CREATE NEW BLOG POST' button using its selector
+ * as it is already there even when the modal dialog box is opened and the
+ * button is not clickable. So we need to wait for the modal dialog box
+ * to close after saving the author bio details, and after that we can click
+ * the 'CREATE NEW BLOG POST' button.
+ */
+
+/* NOTE 2 */
+/** Giving explicit waitForTimeout when clicking 'Unpublish' or 'DELETE' button
+ * in the blog dashboard page (for deleting or unpublishing any blog post) as we
+ * need to wait for the small transition to complete.
+ * We cannot wait for the particular element using its selector because on
+ * clicking the options button (represented by three dots) that button in
+ * the overlay in instantly loaded in the DOM and also its DOM selector.
+ * But the transition of the overlay is not completed and the unpublish
+ * or delete button is not clickable until the transition is completed.
+ * So we need to wait for that small transition to complete
+ * (just 100millisecond is enough). */
+
+
+// TODO(#16552): user-story(B8 cell) left as blocked by this issue.
```

Comment:
Put this at the user story level (the top, "product manager" level of the tests) in the place where it would be if the functionality was working fine.

Reply:
done